reckon-gradle/src/main/java/org/ajoberstar/reckon/gradle/ReckonPlugin.java

```diff
@@ -15,13 +15,20 @@
  */
 package org.ajoberstar.reckon.gradle;
 
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
 import com.google.common.base.Supplier;
 import com.google.common.base.Suppliers;
 import org.ajoberstar.grgit.Grgit;
 import org.gradle.api.Plugin;
 import org.gradle.api.Project;
+import org.gradle.api.Task;
 
 public class ReckonPlugin implements Plugin<Project> {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Document this public class by adding an explicit description. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.ajoberstar.com/coding_rules#rule_key=squid%3AUndocumentedApi)